library/mlops/tracking.py

```diff
@@ -168,6 +168,12 @@ def start_iteration(iteration_name: str, project_id: str = None, experiment_id:
     # TODO: add exceptions when one of id = None or id does not exists
     # In general, we can create separate folder exceptions, same as it's done in back-end routers,
     # because we need to also add some exceptions for further functions
+
+    if not project_id:
```

Comment:
additionaly, we can use these expections in previous functions, which use `active_project_id` or `active_experiment_id` to check if it's not None, for example in `create_experiment()` function may occur an exception

Reply:
Let's also change outputs of `set_active_project` and `set_active_experiment` from raw dictionary to some pretty-print string like : "Active project set to: {project_id}"